packages/server/src/api/queueRoutes.ts

```diff
@@ -122,25 +122,23 @@ export const queueRoutes: ServerRoute[] = [
 
 // for some reason, JOI.allow(null) means the property has to exist, but can be null
 
-async function questionModelToQuestion(qm: QuestionModel): Promise<Question> {
+function questionModelToQuestion(qm: QuestionModel): Question {
   return {
-    creator: await userModelToUserPartial(await (await qm.creator).user),
+    creator: userModelToUserPartial(qm.creator),
     id: qm.id,
     createdAt: qm.createdAt,
     status: parseStatus(qm.status),
     text: qm.text,
     // qm.taHelped: types says is nonnullable, but it is nullable
-    taHelped:
-      (await qm.taHelped) &&
-      (await userCourseModelToUserPartial(await qm.taHelped)),
+    taHelped: qm.taHelped && userModelToUserPartial(qm.taHelped),
```

Comment:
Does this typecheck? When I wrote this code, this errored, because qm.taHelped wasn't a `UserModel`.